utils/prepare_test_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -o errexit
+
+# Setup virtualenv
+virtualenv venv
+source venv/bin/activate
+
+# Ensure you have the latest version of pip
+pip install --upgrade pip
+
+# Install packs requirements
+pip install -r requirements.txt
+pip install -r requirements-test.txt
+
+# Checkout and install st2 requirements
+git clone https://github.com/StackStorm/st2.git --depth 1 --single-branch --branch v1.5.1 ./st2
```

Comment:
1. Because I botched it. I'll fix that.

2. We need to check it out, because installing and checking out the source serves different purposes. The installation makes it easier to develop things as it means you get a full st2 install available to play around with. The checkout of the source is necessary as it contains testing code which is not part of the packages that gets installed by the install script.